### PR TITLE
Reorganize docker images to use the same tag

### PR DESCRIPTION
**Pull request by sanjay990**

### CODE REVIEW

**Comment by sanjay990** [comment]
@ebd2 @ArturGajowy

File: presto-product-tests/conf/docker/common/compose-commons.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ function canonical_path() {
 
 source ${BASH_SOURCE%/*}/../../../bin/locations.sh
 
-export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-'teradatalabs/cdh5-hive:4'}
```
Comment (ArturGajowy) [inline presto-product-tests/conf/docker/common/compose-commons.sh:19]: please keep the `HADOOP_MASTER_IMAGE` parametrization in place - it's there to enable testing with `hdp` image in the future. So it'd be sth like

```
export DOCKER_IMAGE_VERSION=7a742e2
export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-"teradatalabs/cdh5-hive:$DOCKER_IMAGE_VERSION"}
```

Comment (ArturGajowy) [reply presto-product-tests/conf/docker/common/compose-commons.sh:19]: also rename the variable to `DOCKER_IMAGES_VERSION` (plural)

Comment (sanjay990) [reply presto-product-tests/conf/docker/common/compose-commons.sh:19]: @ArturGajowy : What's the advantage of defining the variable
export HADOOP_MASTER_IMAGE=${HADOOP_MASTER_IMAGE:-"teradatalabs/cdh5-hive:$DOCKER_IMAGE_VERSION"}
We don't have that for kerberos images

Comment (ArturGajowy) [inline presto-product-tests/conf/docker/common/compose-commons.sh]: after giving it some thought I think we shouldn't introduce DOCKER_IMAGE_VERSION parametrization to the compose files. I wouldn't like the `compose-commons.sh` to grow to another `run-on-docker.sh` (BTW in Warsaw we jokingly call it 'dockto'...). So, if the only reason for introducing the `DOCKER_IMAGE_VERSION` variable is avoiding duplication, I'd say the cost of more complexity in compose-commons might not be worth it.

OTOH Changing the version used from sequence number ('4') to git hash looks like a good change and will enable for seamless search-and-replace the next time we need to update the image version. After replacing, the change can be further validated by running:
```
$ ./presto-product-tests/conf/docker/singlenode/compose.sh config | grep image
    image: teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f
    image: teradatalabs/cdh5-hive:4
    image: teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f
    image: mysql
    image: postgres
    image: teradatalabs/centos6-java8-oracle:jdk1.8.0_92-80a011f
```

**Review by ArturGajowy — CHANGES_REQUESTED** [review]
LGTM % comments

File: presto-product-tests/conf/docker/common/compose-commons.sh (hunk 1 above)
Comment (ArturGajowy) [reply presto-product-tests/conf/docker/common/compose-commons.sh:19]: See the (edited) comment above :) We don't have the hdp-master-and-slave-in-one image (the dh5-hive equivalent) yet, but once we have it, no changes are needed in presto itself - just setting the env variable in `presto-checks` or wherever we run

**Review by ArturGajowy — APPROVED** [review]

File: presto-product-tests/conf/docker/common/compose-commons.sh (hunk 1 above)
Comment (sanjay990) [reply presto-product-tests/conf/docker/common/compose-commons.sh:19]: :+1:

**Review by ebd2 — APPROVED** [review]
LGTM

**Pull request merged**